avocado/core/sysinfo.py

```diff
@@ -491,30 +124,41 @@ def _get_syslog_watcher():
                     "/var/log/system.log"]
         for logpath in logpaths:
             if os.path.exists(logpath):
-                return LogWatcher(logpath)
+                try:
+                    return sysinfo.LogWatcher(logpath)
+                except PermissionError as e:
+                    log.debug(e.args[0])
```

Comment:
If I understood correctly, this `try` block is just adding a missing check, like other `try` blocks below. Personally, I would add it on a different commit as I would not consider them part of the refactor but a fix.

Anyway, no problem if you leave them in the same commit.